build fails with Python 3.3.3 on Win7 x64 with VS2010

I have a successful build on this Win 7 x64 system from 2014-01-20. I just tried to do a build using python 3.3.3 x64 from the current pull:

> $ git whatchanged
> commit ee3ca829c6e9ec55a444d2c15ab538c548312976
> Author: Michal Romaniuk michal.romaniuk06@ic.ac.uk
> Date:   Sun Jan 26 18:01:59 2014 +0000

The build fails with a link error complaining about not being able to find "round"

> Creating library build\temp.win-amd64-3.3\Release\skimage\feature\orb_cy.lib and
> object build\temp.win-amd64-3.3\Release\skimage\feature\orb_cy.exp orb_cy.obj :
> error LNK2019: unresolved external symbol round referenced in function 
> __pyx_pf_7skimage_7feature_6orb_cy__orb_loop

This looks like a libc issue, because looking at orb_cy.pyx I see:

> from libc.math cimport sin, cos, round

Since  VS2010 is the compiler for Python 3.3.X on Windows. I was hoping someone had seen this before and can help me fix this.

Thanks,
John

--- a/skimage/feature/orb_cy.pyx
+++ b/skimage/feature/orb_cy.pyx
@@ -9,7 +9,9 @@ import numpy as np
 from skimage import data_dir
 
 cimport numpy as cnp
-from libc.math cimport sin, cos, round
+from libc.math cimport sin, cos
+
+from skimage._shared.interpolation cimport round
 
 POS = np.loadtxt(os.path.join(data_dir, "orb_descriptor_positions.txt"),
                  dtype=np.int8)

--- a/skimage/transform/_hough_transform.pyx
+++ b/skimage/transform/_hough_transform.pyx
@@ -15,9 +15,7 @@ from skimage.draw import circle_perimeter
 cdef double PI_2 = 1.5707963267948966
 cdef double NEG_PI_2 = -PI_2
 
-
-cdef inline Py_ssize_t round(double r):
-    return <Py_ssize_t>((r + 0.5) if (r > 0.0) else (r - 0.5))
+from skimage._shared.interpolation cimport round
 
 
 def _hough_circle(cnp.ndarray img,

